justfile

```diff
@@ -46,8 +42,8 @@ test-cargo-example:
 # Run database tests
 test-cargo-db: 
     just test-db setup
-    DATABASE_URL={{DATABASE_URL}} sqlx migrate run --source ./bento/crates/taskdb/migrations/
-    cd bento && DATABASE_URL={{DATABASE_URL}} RISC0_DEV_MODE=1 cargo test -p taskdb
+    git clone https://github.com/risc0/risc0.git && cd risc0/bento
+    DATABASE_URL={{DATABASE_URL}} sqlx migrate run --source ./crates/taskdb/migrations/
```

Comment:
Since Bento is gone, we don't want to do these migrations. We do still need to setup a DB for testing the order-stream, otherwise we'd remove this entirely.